### PR TITLE
Cleaning up white spaces and python syntax

### PR DESCRIPTION
**Pull request by pepper-jk**
Pretty much what the title says.

There were a lot of unneeded tabs in empty lines. I also changed tabs to 4 spaces instead (hope you don't mind). The other changes remove unneeded brackets and semicolons.

This is the code base of 2 additional pull requests. One bringing Gogs support and one enhancing the GitHub support.

### CODE REVIEW

File: gh-issues-import-ng.py [hunk 1]
```diff
@@ -13,496 +13,528 @@
...
-	INITIALIZING         = "script-initializing"
-	LOADING_CONFIG       = "loading-config"
-	FETCHING_ISSUES      = "fetching-issues"
-	GENERATING           = "generating"
-	IMPORT_CONFIRMATION  = "import-confirmation"
-	IMPORTING            = "importing"
-	IMPORT_COMPLETE      = "import-complete"
-	COMPLETE             = "script-complete"
-	
+    current = ""
```
Comment (leemgs) [inline gh-issues-import-ng.py:18]: Looks good to me.

File: gh-issues-import-ng.py [hunk 2]
```diff
@@ -13,496 +13,528 @@
...
+    COMPLETE = "script-complete"
+
+
 state.current = state.INITIALIZING
 
 http_error_messages = {}
-http_error_messages[401] = "ERROR: There was a problem during authentication.\nDouble check that your username and password are correct, and that you have permission to read from or write to the specified repositories."
-http_error_messages[403] = http_error_messages[401]; # Basically the same problem. GitHub returns 403 instead to prevent abuse.
-http_error_messages[404] = "ERROR: Unable to find the specified repository.\nDouble check the spelling for the source and target repositories. If either repository is private, make sure the specified user is allowed access to it."
+http_error_messages[
```
Comment (leemgs) [inline gh-issues-import-ng.py]: @pepper-jk Why do you update the existing **http_error_messages** statement? what for?

File: gh-issues-import-ng.py [hunk 3]
```diff
@@ -13,496 +13,528 @@
...
+
+    config.set('settings', 'import-comments', str(not args.ignore_comments))
+    config.set('settings', 'import-milestone', str(not args.ignore_milestone))
+    config.set('settings', 'import-labels', str(not args.ignore_labels))
+    config.set('settings', 'import-pulls', str(not args.ignore_pulls))
+
+    config.set('settings', 'import-open-issues', str(args.import_all or args.import_open))
+    config.set('settings', 'import-closed-issues', str(args.import_all or args.import_closed))
+
+    # Make sure no required config values are missing
```
Comment (leemgs) [inline gh-issues-import-ng.py:141]: @pepper-jk I want you to separate this change from this PR. I recommend that you submit new PR with this change (e.g., api_url) . :)

Comment (pepper-jk) [reply gh-issues-import-ng.py:141]: What change do you mean? I do not change the API URLs.

File: gh-issues-import-ng.py (hunk 2 above)
Comment (pepper-jk) [reply gh-issues-import-ng.py]: This was an automated change, I was not aware of. It was probably made to meet the line ending specifications of the [pep8 style guide](https://www.python.org/dev/peps/pep-0008/), but for whatever reason it did not split the string to complete this task. I'll change that.

Comment (pepper-jk) [reply gh-issues-import-ng.py]: Fixed.

File: gh-issues-import-ng.py (hunk 3 above)
Comment (pepper-jk) [reply gh-issues-import-ng.py:141]: Any updates?

**Comment by pepper-jk** [comment]
Any Updates?

**Comment by leemgs** [comment]
LGTM

File: gh-issues-import-ng.py (hunk 2 above)
Comment (leemgs) [reply gh-issues-import-ng.py]: Thanks.

File: gh-issues-import-ng.py (hunk 3 above)
Comment (leemgs) [reply gh-issues-import-ng.py:141]: :100:

**Pull request merged**